layouts/partials/full-blog-body.html

```diff
@@ -1,7 +1,7 @@
     <div class="row">
       <article class="col-md-12 blogpost">
         <header>
-          <a href="{{ .Permalink }}"><h1 id="{{ .File.TranslationBaseName }}">{{ .Title }}</h1></a>
+          <a href="{{ .Permalink }}"><h2 id="{{ .File.TranslationBaseName }}">{{ .Title }}</h2></a>
```

Comment:
```suggestion
          <a href="{{ .Permalink }}"><h2 id="{{ .File.TranslationBaseName }}" class="h1">{{ .Title }}</h2></a>
```

Adds h1 styling to h2 heading.